cmd/resolve-version/main.go

```diff
@@ -47,6 +48,53 @@ type matchResult struct {
 	matched            bool
 }
 
+func main() {
+	if len(os.Args) < 3 {
+		printUsage()
+		os.Exit(0)
+	}
+	binary := os.Args[1]
+	versionRequirement := os.Args[2]
+
+	if binary == "yarn" {
+		objects, err := listS3Objects("heroku-nodebin", "yarn")
+		if err != nil {
+			fmt.Println(err)
```

Comment:
One thing to consider in the future is that fmt prints to stdout, while errors typically go to stderr. [log.Fatal*](https://golang.org/pkg/log/#Fatal) both write to stderr and call os.Exit(1). log can be configured via [SetFlags](https://golang.org/pkg/log/#SetFlags) to not include any extra information: https://play.golang.org/p/WL6rVZiotU-